core/cloudflow-operator/src/test/scala/cloudflow/operator/runner/FlinkRunnerSpec.scala

```diff
@@ -145,6 +145,42 @@ class FlinkRunnerSpec extends WordSpecLike with OptionValues with MustMatchers w
       )
     }
 
+    "read labels for pod and adding them to jobmanager and taskmanager" in {
```

Comment:
You need 3 more tests, for values defined in `kubernetes.pods.task-manager.labels`, `kubernetes.pods.job-manager.labels` and both of them defining different values. In that case the job-manager should get different labels from the task-manager pod.

Reply:
Done. Very good call actually